public/app/features/backup/components/KeysBlock/KeysBlock.tsx

```diff
@@ -16,7 +16,9 @@ export const KeysBlock: FC<KeysBlockProps> = ({ accessKey, secretKey }) => {
       </div>
       <div data-qa="secret-key">
         <span className={styles.keyLabel}>{Messages.secretKey}</span>
-        <SecretToggler secret={secretKey} />
+        <span className={styles.secretTogglerWrapper}>
+          <SecretToggler minified secret={secretKey} />
```

Comment:
Nit: I don't really like the name "minified", what about something like "small"? I think it's clearer

Reply:
👍